Ladybird/FontPluginQt.cpp

```diff
@@ -11,8 +11,6 @@
 #include <LibCore/StandardPaths.h>
 #include <LibGfx/Font/Emoji.h>
 #include <LibGfx/Font/FontDatabase.h>
-#include <QFont>
-#include <QFontInfo>
```

Comment:
ConsoleWidget asks for the system default font

https://github.com/SerenityOS/serenity/blob/b5a728ae5f62c4fabf69dc7fb39883fcb210a69e/Ladybird/ConsoleWidget.cpp#L50C4-L50C4

As does Tab

https://github.com/SerenityOS/serenity/blob/b5a728ae5f62c4fabf69dc7fb39883fcb210a69e/Ladybird/Tab.cpp#L186

Should we have those widgets query our font database as well, or are they find to query things in that way as they're Qt Widgets?

Reply:
I think it's fine for widgets to ask for widget-appropriate fonts. The problem is that many of the CSS generic fonts like `serif`, `cursive`, `fantasy` etc have no corresponding "system font".